src/test/ui/span/borrowck-call-is-borrow-issue-12224.stderr

```diff
@@ -17,13 +17,18 @@ error: cannot borrow immutable borrowed content `*f` as mutable
 35 | fn test2<F>(f: &F) where F: FnMut() {
    |                -- use `&mut F` here to make mutable
 36 |     (*f)(); //~ ERROR: cannot borrow immutable borrowed content `*f` as mutable
-   |     ^^^^
+   |     ^^^^ cannot borrow as mutable
 
 error: cannot borrow immutable `Box` content `*f.f` as mutable
   --> $DIR/borrowck-call-is-borrow-issue-12224.rs:44:5
    |
+18 |     f: Box<FnMut() + 'a>
+   |     -------------------- this field should be `&mut`
```

Comment:
What should be the appropriate message for this case? Nothing if the type is not a ref?